src/status_im/contexts/wallet/add_account/create_account/import_private_key/add_key_pair_name/view.cljs

```diff
@@ -0,0 +1,77 @@
+(ns status-im.contexts.wallet.add-account.create-account.import-private-key.add-key-pair-name.view
+  (:require
+    [quo.core :as quo]
+    [react-native.core :as rn]
+    [status-im.common.floating-button-page.view :as floating-button-page]
+    [status-im.common.validation.general :as validators]
+    [status-im.contexts.wallet.add-account.create-account.import-private-key.add-key-pair-name.style :as
+     style]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(def keypair-name-max-length 15)
+(def keypair-name-min-length 5)
+
+(def error-messages
+  {:too-long     (i18n/label :t/key-name-error-length)
+   :too-short    (i18n/label :t/key-name-error-too-short {:count keypair-name-min-length})
+   :emoji        (i18n/label :t/key-name-error-emoji)
+   :special-char (i18n/label :t/key-name-error-special-char)})
+
+(defn view
+  []
+  (let [customization-color (rf/sub [:profile/customization-color])
+        key-pair-name       (rf/sub [:wallet.import-private-key/key-pair-name])
+        [error set-error]   (rn/use-state)
+        on-change-text      (rn/use-callback
```

Comment:
we can define this outside the `view` as a defn and we don't need to use use-callback then

Reply:
Ok, but in this case I have to pass `set-error` as argument